cypress/e2e/requests/new-request-with-patron-comment.cy.js

```diff
@@ -0,0 +1,52 @@
+import TopMenu from '../../support/fragments/topMenu';
+import NewRequest from '../../support/fragments/requests/newRequest';
+import { DevTeams, TestTypes } from '../../support/dictionary';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import EditRequest from '../../support/fragments/requests/edit-request';
+
+describe('ui-requests: Request: Create a New Request with Patron Comment.', () => {
+  const folioInstances = InventoryInstances.generateFolioInstances();
+  const user = {};
+  const testData = {};
+
+  before(() => {
+    cy.loginAsAdmin();
```

Comment:
There are permissions specified for current test:
```
Logged in as user with next permissons:

Requests: All permissions
User is on the "Requests" app
```

Temp user need to be created for the test with mentioned permissions set and deleted after the execution